portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -801,13 +796,22 @@ else if ("always".equals(showHelpGlobal))
 				
 		boolean showPresence;
 		String globalShowPresence = ServerConfigurationService.getString("display.users.present","true");
-				
+		boolean hidePresenceTool = false;
+
+		try {
+			hidePresenceTool = site.getProperties().getBooleanProperty("presenceTool");
+		} catch (EntityPropertyNotDefinedException e) {
+			e.printStackTrace();
+		} catch (EntityPropertyTypeException e) {
+			e.printStackTrace();
```

Comment:
Logger needed, but this probably should be logged as it's not expected, however a good log message (including site ID) is better than a stack trace.